active/CommandBar/new-command-bar-features.md

```diff
@@ -0,0 +1,269 @@
+Currently, a CBF with secondary commands always shows a […] button that allows the user to collapse the CBF into just the top bar (primary commands). There's currently no way to have a CBF with secondary commands that is always expanded when invoked. 
+
+When the CBF is invoked via touch, only the primary commands are shown (with the option for the user to expand the list of secondary commands). The developer is currently not able to control how the CBF appears when invoked, and cannot control whether it can be collapsed or not. 
+
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+### Feature #1. Built-in styles to support adding SplitButton to CommandBar and CommandBarFlyout
+WinUI will provide built in styles for SplitButton that will allow them to conform with the look and feel of an AppBarButton. These styles will change properties such as the SplitButton's height/width, font size, color, interactions (color changes), and more. 
+
+There will be two styles provided: `CommandBarSplitButtonStyle` and `CommandBarFlyoutSplitButtonStyle`. These styles can be accessed as ThemeResources in any WinUI app. 
+
+### Feature #2. RadioMenuFlyoutSubItem
+RadioMenuFlyoutSubItem is a new class that represents a "parent" RadioMenuFlyout item. It is very similar to the [MenuFlyoutSubItem](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyoutsubitem?view=winrt-19041) class.
+
+All child items of a RadioMenuFlyoutSubItem should be of type [RadioMenuFlyoutItem](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.radiomenuflyoutitem?view=winui-2.5) (although this is not enforced by the API). If a child item is selected, the parent item will show as selected. All child items should have the same GroupName as siblings of the parents. 
+
+The parent (RadioMenuFlyoutSubItem) cannot be selected. Thus it does not have GroupName or IsChecked properties. It can only achieve selection (and show the selection indicator) if it is bubbled up from one of its child items. Therefore, only one item out of the child items and other top-level RadioMenuFlyout items in the same Group can be selected at a time. 
+
+If a child item becomes selected, the selection indicator will bubble up to its parent when its MenuFlyout is closed. If a child is selected and the MenuFlyout is open, both the parent and child will show the selection indicator (similar to a hierarchical NavigationView).
+
+These are the three main visual states for this new API, using the same Xaml Controls Gallery sample that was previously shown:
+
+*Child item selected, flyout open*
+
+![Child item selected, flyout open](images/commandbar2.PNG)
+
+*Child item selected, flyout closed*
+
+![Child item selected, flyout open](images/commandbar3.PNG)
```

Comment:
This doesn't appear to match the picture; in the picture the child item isn't selected (Bluetooth is).

Reply:
I believe this should be solved when the pictures are uploaded in my latest commit.